### DISCUSSION TITLE
【岛屿】与【河流】，对CH类平台未来模式的一些畅想。

### DISCUSSION THREAD

**Discussion by cyberlinden** [qihuju/qihuju.github.io#23] [Ideas]
感谢TZ的邀请，我在这里献丑把几天前某个CH房间中的一些火花分享给奇忽局的大家：

首先我想把自己这一个多月CH用户体验层面Pro&Con总结一下：

**Pros：**

- 听觉驱动信息带宽占用较低，媒体温度冷热自由切换，方便用户multi-tasking场景使用。
- 二级传播与统计传播的自由切换，moderator,speaker与listener之间轻松切换。
- 放弃价值判断，以无为态度成为其他各社交平台之外的留白异数。
- 以话题为导向，打破圈层壁垒。非常适合多学科产学研各环节低成本分享。跨越【人是社会关系的总和】而进入【人是社会交互的总和】的新阶段。平等交流，鼓励【涌现】

**Cons：**

- 用户分享与获取的成本不对称，根本上可能是对内容分享者社交回报积累的支持不足。
- 听后即焚的思想火花轻易地水土流失，每一个房间都是无法两次踏入的河流。

以上的可贵与可惜都来自CH交互模式提供的【不确定性】。

在交流中，我意识到此类平台下一步的潜力可能在于构建一种兼具【不确定性】与【确定性】的二元模式：

- 保持和优化现有的chatroom模式提供【不确定性】以激发【涌现】和思想的火花。
- 类Podcast播客功能提供【确定性】以沉淀积累火花以及复盘。这个体验层可以作为用户Bios的延伸，以内容创造者/Speaker和club为单元形成内容的【岛屿】和【山峰】。

可以鼓励每位用户单人录制内容的同时在全体参与者授权的前提下用低学习成本的用户界面记录编辑以及剪辑room中的讨论火花。产品设计维度的体验可以在声音的编辑转译层面对标Insta中的视频图片滤镜。应用Ai语音识别的文本化时间轴也可以是语音内容的另一个附加feature。

岛屿的存在可以在很大程度上增加用户个体的社交回报，有效降低传播两端的成本差。从收听者的角度也可以迅速筛选出符合自己兴趣的内容创造者/群并保有极大的用户粘性。二元模式形成激励闭环，留下一片山河相望的千高原，也会为未来的内容变现留下更多想象空间。

以上

天舒

**Comment by Lupingyue** [comment]
这段文字使用【岛屿】、【河流】、【山峰】、【千高原】的隐喻，本身就是一篇充满媒介分析、哲学思辨、人类交互远景畅想的美文，赞。
其中的信息量非常大，可以分岔出来讨论的条目很多，也跟 @MRchildNEO @askender @Tz-H @MingxuanXie 等的思考和实践「脑图、语音切片、e-flux、GitHub讨论等」有诸多呼应，先消化一下。
文中提到的很多思考 @askender 使用“程序员思维“和“程序员语言”一直在持续思考、讨论，甚至给出很多可执行方案。非常期待 @askender 对 @cyberlinden 这段文字进行反馈。

**Comment by Lupingyue** [comment]
> 在交流中，我意识到此类平台下一步的潜力可能在于构建一种兼具【不确定性】与【确定性】的二元模式：
> -保持和优化现有的chatroom模式提供【不确定性】以激发【涌现】和思想的火花。
> -类Podcast播客功能提供【确定性】以沉淀积累火花以及复盘。这个体验层可以作为用户Bios的延伸，以内容创造者/Speaker和club为单元形成内容的【岛屿】和【山峰】。

【不确定性】与【确定性】的二元模式，实际上目前的讨论和实践中有两个回应：
1， @askender 在讨论中提到的：[多样化的社区参与形式： 目前有 ch语音&脑图线，和卡牌游戏线、github&wechat讨论&网站线（所谓某种孵化器）](https://github.com/qihuju/qihuju.github.io/discussions/5#discussioncomment-569301)
2， @Tz-H 透过奇忽局网站及GitHub社区搭建、脑图创作、录音房的语音回放、文档下载等方式完成了对clubhouse阅后即焚遗憾的部分补偿

另外还有很多碎片化的讨论，也在持续进行。

**Comment by Tz-H** [comment]
@cyberlinden 

一个Markdown格式上的小提醒：

`-用户分享与获取的成本不对称，根本上可能是对内容分享者社交回报积累的支持不足。`

调整为

`- 用户分享与获取的成本不对称，根本上可能是对内容分享者社交回报积累的支持不足。`

即在`-`和后续文字之间添加一个空格，即可激活Markdown本身的列表功能

**Reply by cyberlinden** [reply]
感谢🙏指导！第一次用Github哈哈

**Reply by Tz-H** [reply]
稍微再微调了一下主贴的格式
